app.js

```diff
@@ -528,7 +528,12 @@ app.post('/dataextension/add/', async function (req, res, next){
 		res.status(201).send(JSON.stringify(newPushKey));
 	} catch(err) {
 		console.dir(err);
-		const error_message = err.response?.data?.additionalErrors[0]?.message;
+		let error_message;
+		if (err.response?.data?.additionalErrors){
+			error_message = err.response?.data?.additionalErrors[0]?.message;
+		} else if (err.response?.data?.errors){
+			error_message = err.response?.data?.errors[0]?.message;
+		}
```

Comment:
It could be worth pulling out this error handling logic into a separate function, so that it isn't repeated throughout the app.
Something like:
```suggestion
		const error_message = TryExtractErrorMessage(err);
```